scripts/install-gc-attachments.sh

```
@@ -0,0 +1,68 @@
+  echo "[install] GC_SKIP_LOAD=1: skipping launchctl (generate/install only)."
+  exit 0
+fi
+
+# Stop existing job if loaded (idempotent reload)
+launchctl bootout "gui/$GUI_UID/$LABEL" 2>/dev/null || true
+
+echo "[install] Loading job..."
+launchctl bootstrap "gui/$GUI_UID" "$PLIST"
+
+echo "[install] Verifying..."
+launchctl print "gui/$GUI_UID/$LABEL" >/dev/null 2>&1 && echo "[install] loaded: $LABEL" || {
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)